docs/getting-started-guides/kubeadm.md

```diff
@@ -121,104 +138,126 @@ kubeadm to tell it what to do.
+will then download and install the cluster database and "control plane"
 components. This may take several minutes.
 
+You can't run `kubeadm init` twice without tearing down the cluster in between,
+see [Tear Down](#tear-down).
+
 The output should look like:
 
 ```
-[kubeadm] WARNING: kubeadm is in alpha, please do not use it for production clusters.
+[kubeadm] WARNING: kubeadm is in beta, please do not use it for production clusters.
+[init] Using Kubernetes version: v1.6.0
+[init] Using Authorization mode: RBAC
 [preflight] Running pre-flight checks
-[init] Using Kubernetes version: v1.5.1
-[tokens] Generated token: "064158.548b9ddb1d3fad3e"
-[certificates] Generated Certificate Authority key and certificate.
-[certificates] Generated API Server key and certificate
-[certificates] Generated Service Account signing keys
-[certificates] Created keys and certificates in "/etc/kubernetes/pki"
-[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+[preflight] Starting the kubelet service
+[certificates] Generated CA certificate and key.
+[certificates] Generated API server certificate and key.
+[certificates] API Server serving cert is signed for DNS names [kubeadm-master kubernetes kubernetes.default kubernetes.default.svc kubernetes.default.svc.cluster.local] and IPs [10.96.0.1 10.138.0.4]
+[certificates] Generated API server kubelet client certificate and key.
+[certificates] Generated service account token signing key and public key.
+[certificates] Generated front-proxy CA certificate and key.
+[certificates] Generated front-proxy client certificate and key.
+[certificates] Valid certificates and keys now exist in "/etc/kubernetes/pki"
 [kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/admin.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/kubelet.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/controller-manager.conf"
+[kubeconfig] Wrote KubeConfig file to disk: "/etc/kubernetes/scheduler.conf"
 [apiclient] Created API client, waiting for the control plane to become ready
-[apiclient] All control plane components are healthy after 61.317580 seconds
+[apiclient] All control plane components are healthy after 16.772251 seconds
 [apiclient] Waiting for at least one node to register and become ready
-[apiclient] First node is ready after 6.556101 seconds
-[apiclient] Creating a test deployment
+[apiclient] First node is ready after 5.002536 seconds
 [apiclient] Test deployment succeeded
-[token-discovery] Created the kube-discovery deployment, waiting for it to become ready
-[token-discovery] kube-discovery is ready after 6.020980 seconds
+[token] Using token: <token>
+[apiconfig] Created RBAC rules
 [addons] Created essential addon: kube-proxy
 [addons] Created essential addon: kube-dns
 
 Your Kubernetes master has initialized successfully!
 
+To start using your cluster, you need to run (as a regular user):
```

Comment:
as a regular user -> as a regular user on the same host

Also, maybe we could hint to people that they can also scp or otherwise copy this file off the master and place it in `~/kube/config` to have `kubectl` automatically work on that host (assuming the master IP is routable, right, cuz that gets written into the kubeconfig file?). I think that will probably be more common than UNIX user accounts on the master.